alert-combiner-py/src/blockchain_indexer_service.py

```diff
@@ -38,6 +39,25 @@ def get_etherscan_url(chain_id):
             return "https://api.snowtrace.io"
 
         raise Exception("Chain ID not supported")
+    
+    @staticmethod
+    def get_first_block_number(chain_id):
```

Comment:
is this going to pose a problem as time progresses? Should these offsets from the current block number and get updated regularly?

Reply:
By the way, I noticed that the endBlockNumber is smaller than Arbitrum's latest, so I changed that (by adding an extra 9 to the 999...99)